atc/engine/builder/delegate_factory_test.go

```diff
@@ -47,12 +48,12 @@ var _ = Describe("DelegateFactory", func() {
 	Describe("GetDelegate", func() {
 		var (
 			delegate   exec.GetDelegate
-			info       exec.VersionInfo
+			info       runtime.VersionResult
```

Comment:
VersionResult -- to me this means the data fields are populated by a Get or Check. is that accurate? we should also consider adding some comments to where this type is declared in runtime/types